src/pages/PostsList/components/PostPreview.jsx

```diff
@@ -0,0 +1,45 @@
+import { LikeOutlined } from '@ant-design/icons'
+import { List, Space, Button } from 'antd'
+import React from 'react'
+
+const IconText = ({ icon, text }) => (
+  <Space>
+    {React.createElement(icon)}
+    {text}
+  </Space>
+)
+
+const PostPreview = ({ post }) => {
+  console.log(typeof post.body)
+
+  function countToThreeDots(text) {
+    let counter = 0
+    for (let i = 0; i < text.length; i++) {
+      if (text[i] === '.') {
+        counter++
+        if (counter === 3) {
+          return i + 1
+        }
+      }
+    }
+  }
```

Comment:
Nice! Good practice but quite hard to understand. 
How about:
```suggestion
  function getPositionThirdDot(string) {
    const char = '.'
    const index = 3;
    return string.split(char, index).join(char).length;
  }
```